integration-tests/chopsticks/src/tests/xcm/limitedReserveTransferSpiritnetHydraDxV2.test.ts

```diff
@@ -0,0 +1,52 @@
+import { test } from 'vitest'
+import { sendTransaction, withExpect } from '@acala-network/chopsticks-testing'
+
+import * as SpiritnetConfig from '../../network/spiritnet.js'
+import * as HydraDxConfig from '../../network/hydraDx.js'
+import { KILT, keysAlice } from '../../utils.js'
+import { spiritnetContext, hydradxContext, getFreeBalanceSpiritnet, getFreeBalanceHydraDxKilt } from '../index.js'
+import { getAccountDestinationV2, getNativeAssetIdLocation, getSiblingDestination } from '../../network/utils.js'
+import { checkBalance, createBlock, hexAddress, setStorage } from '../utils.js'
+
+const KILT_ASSET_V2 = { V2: [getNativeAssetIdLocation(KILT)] }
+
+test('Limited Reserve V2 Transfers from Spiritnet Account Alice -> HydraDx', async ({ expect }) => {
+	const { checkEvents, checkSystemEvents } = withExpect(expect)
+
+	// Set storage
+	await setStorage(spiritnetContext, SpiritnetConfig.assignNativeTokensToAccount([keysAlice.address]))
```

Comment:
Should these be part of a `beforeEach` or `beforeAll` hook?

Reply:
I am not sure. My intention was to keep the initial state as minimalistic as possible to ensure it's as generic as possible in every test case. So this was an active design decision. Changing that would also mean, to update the test case for sending funds from hydra -> spiritnet, since I assume there, Alice has no funds.

Reply:
Yeah makes sense.